cogs/runescape_commands.py

```diff
@@ -0,0 +1,72 @@
+import json
+from pathlib import Path
+
+import voxelbotutils as utils
+import aiohttp
+
+
+API_BASE_URL = 'https://secure.runescape.com/m=itemdb_oldschool/api/'
+ITEM_IDS_PATH = Path().parent.joinpath('config').joinpath('osrs-item-ids.json')
+
+
+class RunescapeCommands(utils.Cog):
+    # https://pastebin.com/raw/LhxJ7GRG (parsed and modified)
+    def __init__(self, bot):
+        super().__init__(bot)
+        with open(ITEM_IDS_PATH) as item_ids_file:
+            self.item_ids = json.load(item_ids_file)
+
+    # https://github.com/JMcB17/osrs-blast-furnace-calc
+    @staticmethod
+    def rs_notation_to_int(value_str):
+        multipliers = {
+            'k': 10 ** 3,
+            'm': 10 ** 6,
+            'b': 10 ** 9,
+        }
+
+        value_str = value_str.replace(',', '')
+
+        for multi in multipliers:
+            if value_str.endswith(multi):
+                value_str = value_str.rstrip(multi)
+                value = int(value_str) * multipliers[multi]
+                break
+        else:
+            value = int(value_str)
+
+        return value
+
+    def get_item_value_by_id(self, item_id, return_int=True):
```

Comment:
This needs to be an async function

Reply:
got dang

Reply:
c9a4d63a833e4e4815551dc960ce276d0e50fcac